src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1263,6 +1293,738 @@
 		}
 	},
 
+	"SAMPLE_CONFIG_DB_JSON": {
```

Comment:
I think, this diff is not expected ?

Reply:
@praveen-li, I did `git rebase`, so this diff is expected because there was a `merge conflicts`, because `"SAMPLE_CONFIG_DB_JSON"` keys increased with new fields in `yangTest.json` file.